CONTRIBUTING.md

```diff
@@ -1,80 +1,96 @@
-Contributing to Bitcoin Core
-============================
+# Contributing to Bitcoin Classic
 
-The Bitcoin Core project operates an open contributor model where anyone is welcome to contribute towards development in the form of peer review, testing and patches. This document explains the practical process and guidelines for contributing.
+Bitcoin Classic is hard forking bitcoin to a 2 MB blocksize limit. Please come and join us.
 
-Firstly in terms of structure, there is no particular concept of “Core developers” in the sense of privileged people. Open source often naturally revolves around meritocracy where longer term contributors gain more trust from the developer community. However, some hierarchy is necessary for practical purposes. As such there are repository “maintainers” who are responsible for merging pull requests as well as a “lead maintainer” who is responsible for the release cycle, overall merging, moderation and appointment of maintainers.
+ - [Contribution Guidelines](#contribution-guidelines)
+ - [Communication](#communication)
+ - [Merge Window](#merge-window)
+ - [Consider.it](#considerit)
+ - [Code of Conduct](#codeofconduct)
 
+## <a name="contribution-guidelines"></a> Contribution Guidelines
 
-Contributor Workflow
---------------------
+**Any change to Bitcoin must be respected by 3 groups:**
 
-The codebase is maintained using the “contributor workflow” where everyone without exception contributes patch proposals using “pull requests”. This facilitates social contribution, easy testing and peer review.
+- Developers: which implement changes
+- Miners: which give them computation
+- Users: which give them value
 
-To contribute a patch, the workflow is as follows:
+**We accept pull requests using this rubric:**
 
-  - Fork repository
-  - Create topic branch
-  - Commit patches
+- All pull requests must be within the [merge window](#merge-window).
```

Comment:
Get rid of "All". There are some exceptions to this rule, such as emergency patches, bugfixes to patches that were submitted in the previous pull request, or new features which everyone agrees are super duper helpful and technically simple and stuff. We don't need to make it impossible for us to merge emergency stuff; we just need to make it easy for the maintainers/judicial branch to stuff down that violates the merge window and would be disruptive if merged at that time.